source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -157,5 +157,72 @@ the Query API:
                                       then: "extreme temperature",
                                       else: "no advisory" } } } }
 
+Comparison Operations
+~~~~~~~~~~~~~~~~~~~~~
+
+You can use comparison operations to compare two values. The methods
+described in this section perform a comparison operation on ``MqlValue``
+objects.
```

Comment:
It would help to clarify that all types are of MqlValue. For consistency with the other sections, perhaps something like:

> You can perform a comparison operation on any value of type `MqlValue` using the methods described in this section. All values such as MqlBoolean, Mql...etc. are of type MqlValue.

The term "object" should be avoided, since this might be confused with MqlDocument (which correspond to what the server docs call objects aka documents).

One implication of any type being able to be compared against any other type is that [the comparison order](https://www.mongodb.com/docs/manual/reference/bson-type-comparison-order/#std-label-bson-types-comparison-order) applies (though, that list will be confusing from the point of view of our API, because the API hides several irrelevant items, and we don't use the term "object" but "document" instead).